kolibri/plugins/learn/assets/src/views/card-grid/grid-item/index.vue

```diff
@@ -59,57 +103,123 @@
 
   .card
+    display: inline-block
     width: $card-width
     height: $card-height
-    overflow: hidden
+    border-radius: 2px
     background-color: $core-bg-light
-    border-radius: $radius
+    box-shadow: $card-elevation-resting
+    &:hover, &:focus
+      box-shadow: $card-elevation-raised
 
   .card-thumbnail
-    height: 122px
     position: relative
+    width: $card-width
+    height: $card-thumbnail-height
+    background-size: cover
+    background-position: center
+    background-color: $core-grey
 
-  .text
-    -webkit-line-clamp: 3 // Enhance Chrome, doesn't work on other browsers
-    -webkit-box-orient: vertical // Enhance Chrome, doesn't work on other browsers
-    width: auto
-    max-height: 68px
-    margin-left: 35px
-    overflow: hidden
-    font-size: 0.9rem
-    line-height: 1.2rem
-    text-overflow: ellipsis
-    color: $core-text-default
-
-  .content-icon
+  .card-thumbnail-backup
     position: absolute
-    left: 10px
-    top: 10px
-    font-size: 1.5em
-
-  .card-content
-    padding: 10px
-    position: relative
+    top: 50%
+    left: 50%
+    transform: translate(-50%, -50%)
+    color: $core-text-annotation
+    font-size: ($card-thumbnail-height / 2)
 
-  a
-    text-decoration: none
-
-  .progress-icon-wrapper
+  .card-progress-icon-wrapper
     position: absolute
     top: 0.25em
     right: 0.25em
     width: 1.5em
     height: 1.5em
 
-  .wrapper
-    background-color: $core-bg-light
+  .card-content-icon-background
+    position: absolute
+    top: 0
+    left: 0
+    width: 0
+    height: 0
+    border-style: solid
+    border-width: 3.5em 3.5em 0 0
+    border-top-color: $core-content-topic
+    border-right-color: transparent
+    border-bottom-color: transparent
+    border-left-color: transparent
+
+  .card-content-icon-background-exercise
+    border-top-color: $core-content-exercise
+
+  .card-content-icon-background-video
+    border-top-color: $core-content-video
+
+  .card-content-icon-background-audio
+    border-top-color: $core-content-audio
 
-  .link-block
-    display: block
+  .card-content-icon-background-document
+    border-top-color: $core-content-document
+
+  .card-content-icon-background-html5
+    border-top-color: $core-content-html5
+
+  .card-content-icon-wrapper
+    position: absolute
+    top: 0
+    left: 0
+    padding: 0.25em
+    color: white
+
+  .card-content-icon
+    font-size: 1.25em
+
+  .card-progress-bar-wrapper
+    position: absolute
+    bottom: 0
+    background-color: $core-grey
+    width: 100%
+    height: 5px
+
+  .card-progress-bar
+    height: 100%
+
+  .card-progress-bar-mastered
+    background-color: $core-status-mastered
+
+  .card-progress-bar-progress
+    background-color: $core-status-progress
+
+  .card-text
+    padding: $card-text-padding
+    height: $card-text-height
+    color: $core-text-default
+
+  .card-title, .card-subtitle
+    margin: 0
+
+  .card-title
+    overflow: hidden
+    line-height: ($card-width / (320 / 32))
```

Comment:
320 used here and various other places. What's it representative of? Should make it a variable